pkg/virtctl/guestfs/guestfs.go

```diff
@@ -188,28 +204,41 @@ type K8sClient struct {
 // setImage sets the image name based on the information retrieved by the KubeVirt server.
 func setImage(virtClient kubecli.KubevirtClient) error {
 	var imageName string
-	info, err := virtClient.GuestfsVersion().Get()
+	info, err := ImageInfoGetFunc(virtClient)
 	if err != nil {
 		return err
 	}
+	// Set image name including prefix if available
+	imageName = fmt.Sprintf("%s%s", info.ImagePrefix, defaultImageName)
```

Comment:
@akalenyu does the prefix already include teh `-` or do you need to add it?

Reply:
Should be there